src/controllers/check.your.answers.controller.ts

```diff
@@ -32,8 +32,13 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
     logger.debug(`POST ${config.CHECK_YOUR_ANSWERS_PAGE}`);
 
     const transaction: Transaction = await postTransaction(req.session as Session);
+    logger.info(`Transaction created, ID: ${transaction.id}`);
+
     const overseaEntity: OverseasEntityCreated = await createOverseasEntity(req.session as Session, transaction.id as string);
-    await closeTransaction(req.session as Session, transaction.id as string, overseaEntity.id);
+    logger.info(`Overseas Entity Created, ID: ${overseaEntity.id}`);
+
+    const responseClosedTransaction = await closeTransaction(req.session as Session, transaction.id as string, overseaEntity.id);
+    logger.info(`Transaction Closed,  ${JSON.stringify(responseClosedTransaction)}`);
```

Comment:
mentioned in PR 77 (which this is based on), this should be changed to log just the ID instead of the whole transaction object

Reply:
Thanks Mark for looking into it and as I said, I agree but first we need to change the response coming from the SDK, it will be done in a future story